build_system/src/build.rs

```diff
@@ -26,9 +27,7 @@ impl BuildArg {
                         build_arg.flags.push("--features".to_string());
                         build_arg.flags.push(arg.as_str().into());
                     } else {
-                        return Err(
-                            "Expected a value after `--features`, found nothing".to_string()
-                        );
+                        return Err("Expected a value after `--features`, found nothing".to_string());
```

Comment:
CI is failing because it's not happy with this change. (Wrong formatting)

Reply:
reverted the change
i am gonna commit now and see what happens